lib/marked.js

```diff
@@ -616,6 +615,9 @@ inline.gfm = merge({}, inline.normal, {
     .getRegex()
 });
 
+inline.gfm.url = edit(inline.gfm.url)
+  .replace('email', inline.gfm._extended_email)
+  .getRegex();
```

Comment:
Why was this moved? It looks like the only change to the previous code was `inline._email` changed to `inline.gfm._extended_email`

Reply:
Those two email regexes are similar, but different. It was moved out of the block because I had no way of referencing inline.gfm._extended_email from _inside_ the block.